pkg/apis/extensions/v1beta1/types.go

```diff
@@ -397,6 +397,10 @@ type DeploymentStatus struct {
 	// +optional
 	UpdatedReplicas int32 `json:"updatedReplicas,omitempty" protobuf:"varint,3,opt,name=updatedReplicas"`
 
+	// Total number of ready pods targeted by this deployment.
+	// +optional
+	ReadyReplicas int32 `json:"readyReplicas,omitempty"`
```

Comment:
@smarterclayton any idea why protobuf is failing to be updated here? I even tried to regenerate by moving the field at the end of the struct but with no luck.